### PR TITLE
Add Photography Section and Social Media Logos

### PR DESCRIPTION
**Pull request by maceonthompson**
There are 3 commits but the main addition is a photography section with a few of my photos. Another small addition is an embedded soundcloud link.

### CODE REVIEW

**Comment by maceonthompson** [comment]
Ah so I didn't know that pushing to this branch would also update the pull request - What's best practice here? Do I wait and leave my changes stached locally, do I make separate branches, or is this what people mean when they say they're "blocked" by a review.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,12 +7,39 @@
...
+      <div class = "logos">
+        <iframe allowtransparency="true" 
+          scrolling="no" 
+          frameborder="no" 
+          src="https://w.soundcloud.com/icon/?url=http%3A%2F%2Fsoundcloud.com%2Fmaceo_music&color=white_orange&size=48" 
+          style="width: 48px; height: 48px;">
+        </iframe>
+
+        <a href="https://github.com/maceonthompson">
+          <img src="Logos/GitHubLogo.png" style="width:48px; heigh: 48px;">
```
Comment (kristen-law) [inline portfolio/src/main/webapp/index.html:32]: Typo on height here and below

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,12 +7,39 @@
...
+        photography, reading, gaming. I (of course) make music and code as well, though I do 
+        more of the former for fun.
+      </p>
+      <p>I'm not much of a designer, but I do like to take photographs!</p>
+      <p>Click <a href="photos.html">here</a> to see some of my photography!</p>
+
+      <p>Social media:</p>
+
+      <div class = "logos">
+        <iframe allowtransparency="true"
```
Comment (kristen-law) [inline portfolio/src/main/webapp/index.html:24]: Is there a reason you're using an iframe here instead of just linking an image?

Comment (maceonthompson) [reply portfolio/src/main/webapp/index.html:24]: This was the first link I included, and soundcloud happened to provide a template for it. I could change it to an image link for consistency of code, however.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (maceonthompson) [reply portfolio/src/main/webapp/index.html]: Resolved!

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -7,12 +7,39 @@
...
+        <a href="https://github.com/maceonthompson">
+          <img src="Logos/GitHubLogo.png" style="width:48px; height: 48px;">
+        </a>
+
+        <a href="https://www.linkedin.com/in/maceo-thompson-b86471179/">
+          <img src="Logos/LinkedinLogo.png" style="width:48px; height: 48px;">
+        </a>
+
+        <a href="https://www.instagram.com/thompson_maceo/">
+          <img src="Logos/InstagramLogo.png"  style="width:48px; heigh: 48px;">
```
Comment (kristen-law) [inline portfolio/src/main/webapp/index.html:40]: One more height typo here

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (kristen-law) [reply portfolio/src/main/webapp/index.html:24]: Yeah I think it's best to not use an iframe for something that can be done with an img and hyperlink.

**Pull request merged**